tinywallet/tinywallet/screens.py

```diff
@@ -1649,9 +1649,12 @@ def withDCRD(node):
 
                 self.ctl.connectNode(withDCRD)
 
+            self.connectedLbl.setText("Connected to dcrd")
```

Comment:
Since `"Connected to dcrd"` and `"Connection to dcrd is currently off"` are repeated in `readConfig`, we should either 1) assign these strings to module-level variables, or 2) wrap the calls to `self.connectedLbl.setText` in new `DCRDConfigScreen` methods, e.g. `setDcrdLabelOn` and `setDcrdLabelOff`.